projects/storefrontlib/cms-components/navigation/scroll-to-top/scroll-to-top.module.ts

```diff
@@ -0,0 +1,29 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import {
+  CmsConfig,
+  provideDefaultConfig,
+  ScrollBehavior,
+} from '@spartacus/core';
+import { IconModule } from '../../misc/icon/icon.module';
+import { ScrollToTopComponent } from './scroll-to-top.component';
+
+@NgModule({
+  imports: [CommonModule, IconModule],
+  providers: [
+    provideDefaultConfig(<CmsConfig>{
+      cmsComponents: {
+        ScrollToTopComponent: {
+          component: ScrollToTopComponent,
+          data: {
+            scrollBehavior: ScrollBehavior.SMOOTH,
```

Comment:
We should have default values in the component and require configuration only to override the defaults.